specs-go/config.go

```diff
@@ -615,6 +615,25 @@ type Arch string
 // LinuxSeccompFlag is a flag to pass to seccomp(2).
 type LinuxSeccompFlag string
 
+const (
+	// LinuxSeccompFlagLog is the filter to return actions except
+	// SECCOMP_RET_ALLOW should be logged. An administrator may override this
+	// filter flag by preventing specific actions from being logged via the
+	// /proc/sys/kernel/seccomp/actions_logged file. (since Linux 4.14)
+	LinuxSeccompFlagLog LinuxSeccompFlag = "SECCOMP_FILTER_FLAG_LOG"
+
+	// LinuxSeccompFlagTsync is the filter to synchronize all other threads of
+	// the calling process to the same seccomp filter tree. A "filter tree" is
+	// the ordered list of filters attached to a thread. (Attaching identical
+	// filters in separate seccomp() calls results in different filters from
+	// this perspective.)
+	LinuxSeccompFlagTsync LinuxSeccompFlag = "SECCOMP_FILTER_FLAG_TSYNC"
```

Comment:
With @rata we were discussing whether to remove tsync from the spec. In the context of container runtimes,  the process will call seccomp() before exec() to the entrypoint, so all other threads, if any, will be terminated anyway, so tsync does not make a difference.

Examples:

* AFAIU crun is not multithreaded, so tsync is meaningless
* runc is written in Go so multithreaded, but uses libseccomp-golang that always set tsync: it's not configurable because threads and Go routine are not necessarily tightly coupled.

Reply:
I've added it without thinking if it made any sense as I've copied all the existing flags.  I agree that it is pointless in the OCI specs, and we could drop it.

Reply:
I don't think it should be removed, I think it should be marked as deprecated.

See this: https://github.com/opencontainers/runtime-spec/issues/1077

Reply:
Good, removed it from the code now.